### DISCUSSION TITLE
About sponsorships

### DISCUSSION THREAD

**Discussion by vicary** [serverless-heaven/serverless-webpack#1559] [Team Posts]
Hi team,

It maybe an odd time to ask, but have you guys thought of taking sponsorships for `serverless-webpack`?

My company went through another series of funding, I have two leads running the dev team for me now. I am now seriously looking for things that I like to do.

I wanted to dedicate my time for this project for a while already, every time I look at the open issues kills me a little bit inside...

I think this project is great and has a lot of potentials, but every missing feature takes quite some efforts to happen. Having people sponsoring the project, financially supporting those who is still actively maintaining the project will keep it healthy for a long time.

Before we have consensus on a team level, I'd like to be vocal that I am asking for sponsorships personally.

Please share your thoughts!

cc @j0k3r @HyperBrain @miguel-a-calles-mba

**Comment by miguel-a-calles-mba** [comment]
Hi @vicary, I think enabling sponsorship is a great idea especially it will allow you and others to put more love into the project.

Are you think there would be a shared sponsorship fund and that gets disbursed? Maybe through OpenCollective or something meant for open source software development?

Or, are you think about adding the individual sponsorships inside this project? For example, clicking the sponsorship button will show the link to your BuyMeACoffee page?

**Comment by vicary** [comment]
@miguel-a-calles-mba If it doesn't ring a bell to anyone here, I am flattered to have my sponsorship link in the project. Otherwise, please by all means make it a shared one!

**Comment by j0k3r** [comment]
I'm ok with OpenCollective.
I've requested @HyperBrain to accept a connection between OpenCollective and the serverless-heaven organization on GitHub.
Could you check about it @HyperBrain ?
Thanks 🙏

**Comment by HyperBrain** [comment]
Hi Jérémy,
I am on vacation at the moment, but will check as soon as I am back.
I saw the GitHub connection request but cannot do it from here :)
Looking forward,
Frank

**Comment by vicary** [comment]
Hey thanks @j0k3r and @HyperBrain for making this happen!
